lib/oli_web/templates/layout/page.html.eex

```diff
@@ -0,0 +1,37 @@
+<%= render_layout "delivery.html", assigns do %>
+
+  <%= for script <- @scripts do %>
+    <script type="text/javascript" src="<%= Routes.static_path(@conn, "/js/" <> script) %>"></script>
+  <% end %>
+
+  <div id="wrapper">
+      <!-- Sidebar -->
+      <nav id="sidebar" class="outline">
+        <div class="header d-flex flex-column">
+          <div>
+            <a href="<%= Routes.delivery_path(@conn, :index) %>" class="btn btn-link course-btn">
+              <i class="fa fa-arrow-left" aria-hidden="true"></i><i class="fa fa-book" aria-hidden="true"></i> <%= @summary.title %>
+            </a>
+          </div>
+          <div class="flex-fill"></div>
+        </div>
+        <%= render OliWeb.PageDeliveryView, "_outline.html", conn: @conn, context_id: @context_id, summary: @summary, active_page: @slug %>
+      </nav>
+
+      <!-- Page Content -->
+      <div id="content">
+
+        <%= @inner_content %>
+
+        <%= render OliWeb.PageDeliveryView, "_previous_next_nav.html",
+          conn: @conn, context_id: @context_id, summary: @summary, previous_page: @previous_page, next_page: @next_page %>
+
+      </div>
+  </div>
+
+  <script>
+    OLI.initActivityBridge('eventIntercept');
```

Comment:
It seems wrong to init the bridge here, given that this template isn't responsible for the rendering of the 'eventIntercept' DOM element.  Can we move this init back to `lib/oli_web/templates/page_delivery/page.html.eex` where that DOM element is rendered?

Reply:
Yes, agreed. Fixed in latest commit